lib/bot.js

```diff
@@ -36,6 +36,8 @@ class Bot {
     this.channels = _.values(options.channelMapping);
     this.ircStatusNotices = options.ircStatusNotices;
     this.announceSelfJoin = options.announceSelfJoin;
+    this.ignoreBots = options.ignoreBots;
+    this.ignoreIrcNicks = options.ignoreIrcNicks || [];
```

Comment:
I think a plain snowflake id would be best. The bot owner can right click a user and choose "copy ID" to get the ID (if they can spend the time to set this up and have a Discord app, then they can enable dev mode in the client)